.travis.yml

```diff
@@ -40,15 +40,18 @@ script:
     fi
   - set -o pipefail
   - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
-      ./gradlew -DexcludeTags=ScanCodeTag --stacktrace check | tee log.txt;
+      ./gradlew -DexcludeTags=ScanCodeTag --stacktrace detectCheck classes testClasses jacocoReport | tee log.txt;
```

Comment:
Shouldn't also the `funTestClasses` task be called? Also, does `jacocoReport` depend on the test tasks?

Reply:
You're basically right, but I solved this differently.